lib/autoproj/templates/build_report_helper.erb

```diff
@@ -0,0 +1,21 @@
+<% @first_line = true -%>
+{
+   "build_report" : {
+       "packages": [<% package_list.each do |pkg_name| -%>
+<% if @first_line == false  %>           },    <% end -%>
+<% @first_line = false  %>
+           {
+               "name": "<%= Autobuild::Package[pkg_name].name.to_s %>",
```

Comment:
Within autoproj, use `ws.find_autobuild_package(pkg_name)`. This would be more future-proof. This global registration of packages on `Autobuild::Package` is meant to be phased out eventually.